src/script/page/preferences/accountPreferences/AvatarInput.tsx

```diff
@@ -85,30 +88,40 @@ const AvatarInput: React.FC<AvatarInputProps> = ({selfUser, isActivatedAccount,
     }
   };
 
+  const handleKeyPress = (event: React.KeyboardEvent) => {
+    if (event.key === KEY.ENTER || event.key == KEY.SPACE) {
+      inputRef.current.click();
+    }
+  };
+
   return (
-    <label
-      className="preferences-account-picture-button"
-      htmlFor="self-upload-file-input"
-      title={t('tooltipPreferencesPicture')}
-    >
-      <Avatar className="see-through" participant={selfUser} avatarSize={AVATAR_SIZE.X_LARGE}></Avatar>
-      <FileInput
-        id="self-upload-file-input"
-        data-uie-name="do-select-picture"
-        fileTypes={FILE_TYPES}
-        onFileChange={files => {
-          const newUserPicture = files.item(0);
-
-          setPicture(newUserPicture).catch(error => {
-            const isInvalidUpdate = error.type === UserError.TYPE.INVALID_UPDATE;
-            if (!isInvalidUpdate) {
-              throw error;
-            }
-          });
-        }}
-      />
-      <span className="icon-camera"></span>
-    </label>
+    <div tabIndex={0} role="button" onKeyPress={handleKeyPress}>
```

Comment:
is the screen reader reading the labels from the child when this one is focused or we should provide here an `aria-label`?

Reply:
might be valid point ;)

Reply:
added